app/views/govuk_publishing_components/components/docs/fieldset.yml

```diff
@@ -12,10 +12,10 @@ examples:
       legend_text: 'Do you have a passport?'
       text: |
         <!-- Use the radio component, this is hardcoded only for this example -->
-        <input type="radio" id="default-yes" name="default"t>
```

Comment:
🤦‍♂  We should update these examples over time.

Reply:
Yeah, I know! I did spend a moment wondering if that was deliberate and some rails thing I didn't know about...